test/TestTimeSync.cpp

```diff
@@ -24,7 +25,7 @@ TEST(TimeSync, now) {
 
     time_sync::TimeSync time_sync;
     time_sync.start();  
-    EXPECT_NEAR(time_sync.nowInMilliseconds(), 
+    EXPECT_NEAR(time_sync::nowInMilliseconds() , 
```

Comment:
The nowInMilliseconds is instance method?

Reply:
It is not. It is a static function under the tim_sync namespace. I will rename the time_sync variable to be it clear.